Puppetfile

```diff
@@ -4,12 +4,7 @@ forge "http://forge.puppetlabs.com"
 mod "zack/r10k", '1.0.2'
 
 # Deps for zack/r10k
-# We are tracking stdlib from git because the puppet module tool
-# is getting in the way when we want to upgrade newer than the
-# supported module version
-mod "stdlib",
-        :git => 'git@github.com:puppetlabs/puppetlabs-stdlib.git',
-        :ref => '4.1.0'
+mod "puppetlabs/stdlib", '4.4.0'
```

Comment:
Yeah, unfortunately most of my Jenkins-related open source work is at the bottom of my priorities right now :(